### DISCUSSION TITLE
阅读态显示value

### DISCUSSION THREAD

**Discussion by wjwjwjw** [alibaba/formily#1420] [Q&A 2.x] [answered]
非阅读态时显示正常，改为阅读态时显示value值，是为什么
![image](https://user-images.githubusercontent.com/33014303/118129852-46d8ad80-b42f-11eb-9715-32c1dd24f5c3.png)

![image](https://user-images.githubusercontent.com/33014303/118129907-59eb7d80-b42f-11eb-96da-69f8aec21b9a.png)

![image](https://user-images.githubusercontent.com/33014303/118130123-99b26500-b42f-11eb-9fb1-5342c16dbf47.png)

**Comment by janryWang** [comment, marked as answer]
升级最新版就行了
